Library/Homebrew/completions.rb

```diff
@@ -12,6 +12,14 @@ module Homebrew
   module Completions
     extend T::Sig
 
+    Variables = Struct.new(
+      :aliases,
+      :builtin_command_descriptions,
+      :completion_functions,
+      :function_mappings,
+      keyword_init: true,
```

Comment:
I just learned about `keyword_init: true` two days ago and it's the best thing ever.